include/cuco/static_map.cuh

```diff
@@ -484,6 +470,20 @@ class static_map {
     }
 
    public:
+    /**
+     * @brief Gets slots array.
+     *
+     * @return Slots array
+     */
+    __host__ __device__ pair_atomic_type* get_slots() noexcept { return slots_; }
```

Comment:
Why do these need to be public?

Reply:
Because of the `explicit` constructor for `device_mutable_view` in `deview_view`, the information needs to be accessed from an object of the former. Since it's not in the context of the former's class anymore, protected access won't work, right?

Reply:
they both inherit from `device_view_base`, so shouldn't these be accessible if they are protected?

Reply:
I don't believe so. `device_view` hasn't been constructed yet, so those methods don't exist for it. And it receives an instance of `device_mutable_view`, and for those only `public` methods should be callable. Here's the error if I put these methods in `protected`:

```error: protected function "cuco::static_map<Key, Value, Scope, Allocator>::device_view_base::get_slots() noexcept [with Key=int32_t, Value=int32_t, Scope=cuda::__3::thread_scope_block, Allocator=cuco::cuda_allocator<char>]"```